packages/web/src/components/user-card/UserCard.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { SquareSizes } from '@audius/common/models'
+import { Text } from '@audius/harmony'
+import { merge } from 'lodash'
+import { Route, Routes } from 'react-router-dom-v5-compat'
+import { describe, expect, it } from 'vitest'
+
+import { RenderOptions, render, screen } from 'test/test-utils'
+
+import { UserCard } from './UserCard'
+
+function renderUserCard(options?: RenderOptions) {
+  return render(
+    <Routes>
+      <Route path='/' element={<UserCard id={1} size='s' />} />
+      <Route
+        path='/test-user/test-collection'
+        element={<Text variant='heading'>Test Collection Page</Text>}
+      />
+      <Route
+        path='/test-user'
+        element={<Text variant='heading'>Test User Page</Text>}
+      />
```

Comment:
really nice to have this capability for testing. Since we went to the trouble to add it, should we have a test that exercises the link?

Reply:
yeah my b i realized i hadn't pushed all the tests, they are there now though